docs/ops/deployment/kubernetes.md

```diff
@@ -274,44 +341,121 @@ spec:
             path: flink-conf.yaml
           - key: log4j.properties
             path: log4j.properties
-
 {% endhighlight %}
 
-`jobmanager-service.yaml`
+### Job cluster resource definitions
+
+`jobmanager-job.yaml`
 {% highlight yaml %}
-apiVersion: v1
-kind: Service
+apiVersion: batch/v1
+kind: Job
 metadata:
   name: flink-jobmanager
 spec:
-  type: ClusterIP
-  ports:
-  - name: rpc
-    port: 6123
-  - name: blob
-    port: 6124
-  - name: ui
-    port: 8081
+  replicas: 1
   selector:
-    app: flink
-    component: jobmanager
+    matchLabels:
+      app: flink
+      component: jobmanager
+  template:
+    metadata:
+      labels:
+        app: flink
+        component: jobmanager
+    spec:
+      restartPolicy: OnFailure
+      containers:
+        - name: jobmanager
+          image: flink:{% if site.is_stable %}{{site.version}}-scala{{site.scala_version_suffix}}{% else %}latest{% endif %}
+          env:
+          workingDir: /opt/flink
```

Comment:
is this not controlled by the docker image?

Reply:
true, this leftover looks redundant